.github/workflows/size-limit.yml

```diff
@@ -0,0 +1,19 @@
+name: Check bundle size
+on:
+  push:
+    branches:
+      - '**'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      CI_RUN: true
```

Comment:
Why do we need it?

Reply:
I've take this workflow from [antiflasher/apcach](https://github.com/antiflasher/apcach/blob/9dcda387321c2f7417ecb9f92cfa91730bf48244/.github/workflows/size-limit.yml) repo. Actually i think this env is not needed, removed it